docs/user/discover.asciidoc

```diff
@@ -131,18 +131,6 @@ emit("Hello World!");
 +
 [role="screenshot"]
 image:images/hello-field.png[hello field in the document table]
-
-. Create a second field named `customer` that combines customer last name and first initial.
-+
-```ts
-String str = doc['customer_first_name.keyword'].value;
-char ch1 = str.charAt(0);
-emit(doc['customer_last_name.keyword'].value + ", " + ch1);
-```
-. Simplify the the document table by removing `customer_first_name` and `customer_last_name` and adding `customer` in their place.
-+
```

Comment:
`What happens if you forgot to define an important value as a separate field? Or, what if you want to combine two fields and treat them as one? `
So the second part is now missing. Actually this could be switched to a simple useful script like you have a field of $ and want to convert to EUR? this would be both simple and kind of useful ... or calculate Fahrenheit to Celsius? ...

Reply:
I added the content back in.  Will update the example in the next rev.